lib/proto/boltzrpc_pb.js

```
@@ -13491,6 +13535,392 @@ proto.boltzrpc.TransactionHookRequest.prototype.hasVout = function() {
+ * @param {!jspb.BinaryReader} reader The BinaryReader to use.
+ * @return {!proto.boltzrpc.InvoiceCreationHookResponse}
+ */
+proto.boltzrpc.InvoiceCreationHookResponse.deserializeBinaryFromReader = function(msg, reader) {
+  while (reader.nextField()) {
+    if (reader.isEndGroup()) {
+      break;
+    }
+    var field = reader.getFieldNumber();
+    switch (field) {
+    case 1:
+      var value = /** @type {string} */ (reader.readString());
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)